### ISSUE TITLE
Promptly + Connect = POWER UP ACTIVATED

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/connect#29]
[Promptly](https://github.com/codeforamerica/promptly) texts a few hundred (right?) SF CalFresh clients each month who are at risk of getting discontinued. Lots of these clients immediately call the HSA service center to figure out watsup and presumably fix something with their case. BUT it's looking like lots of these calls don't make it very far. Here's the distribution of call duration:
![eep](https://cloud.githubusercontent.com/assets/2533112/4945904/2603d46c-660d-11e4-86f3-8d62270ca91d.png)

This may partially explain why Promptly doesn't seem to be reducing churn much if at all. One possible reason the phone calls are so short (and presumably so useless) is that people either don't want to go through the phone tree, hear the hold time and hangup, or start waiting on hold but don't make it to the end. We can solve ALL of these problems with **DRUM ROLL** Connect (per @ttmp's idea)!

I _think_ all this would involve is changing the voice_url for the Promptly production number to point to the Connect app. @daguar is that right? Then any incoming calls would hear Dave's pretty voice saying they can hang up and get on with their day.

Issues:
- Testing: We have 0 active users for Connect so we would need to test it (preferably) with real users first. We'd like to start with a few CBO/advocates before giving it to clients.
- Language: Connect only goes to the English line right now. I don't remember if Promptly prod has one phone number or unique numbers by language? If it's one phone # across languages this might be an issue. @andyhull can you confirm?
- What am I missing?

cc @rduecyg and @RebeccaCoelius
